omod/src/main/webapp/resources/htmlforms/015-HMIS_ACP_ExposedInfantClinicalChart-EncounterPage.xml

```diff
@@ -0,0 +1,422 @@
+                                <obs conceptId="99080 " answerConceptIds="163006,163007" style="radio" class="horizontal"/>
+                            </div>
+                        </div>
+                        
+                        <h4 class="card-title mt-4">Infant feeding</h4>
+                        <div class="row">
+                            <div class="col-md-4">
+                                <label>Feeding Code</label>
+                                <obs conceptId="99451" answerConceptIds="5526,99089,6046,99791,99792,99793"
+                                    answerLabels="EBF - Exclusive Breast Feeding,RF - Replacement Feeding (never breastfed),MF - Mixed Feeding (below 6 months) ,CF - Complimentary Feeding (above 6 months),W - Wean from breastfeeding,NLB - No Longer Breastfeeding"/>
+                            </div>
+                        </div>
+                        
+                        <h4 class="card-title">Immunisation Codes</h4>
+                        <div class="row">
+                            <div class="col-md-3">
+                                <obs conceptId="162839" answerConceptId="886" answerLabel="1-BCG"/>
+                                <obs conceptId="162839" answerConceptId="162831" answerLabel="2-Polio-0"/>
+                                <obs conceptId="162839" answerConceptId="162832" answerLabel="3-Polio-1"/>
+                                <obs conceptId="162839" answerConceptId="162833" answerLabel="4-Polio-2"/>
+                            </div>
+                            <div class="col-md-3">
+                                <obs conceptId="162839" answerConceptId="162834" answerLabel="5-Polio-3"/>
+                                <obs conceptId="162839" answerConceptId="1422" answerLabel="6-IPV"/> 
+                                <obs conceptId="162839" answerConceptId="162835" answerLabel="7-DPT-HepB + Hib1"/>
+                                <obs conceptId="162839" answerConceptId="162836" answerLabel="8-DPT-HepB + Hib2"/>
+                            </div>
+                            <div class="col-md-3">
+                                <obs conceptId="162839" answerConceptId="163014" answerLabel="9-PCV 1"/> 
+                                <obs conceptId="162839" answerConceptId="163015" answerLabel="10-PCV 2"/> 
+                                <obs conceptId="162839" answerConceptId="163016" answerLabel="11-PCV 3"/> 
+                                <obs conceptId="162839" answerConceptId="63"     answerLabel="14-Measles"/>
+                            </div>
+                            <div class="col-md-3">
+                                <obs conceptId="162839" answerConceptId="99356"  answerLabel="16-Vitamin A"/>
+                                <obs conceptId="162839" answerConceptId="162838" answerLabel="17-De-Worming"/>
+                                <span id="immunization-not-done">
+                                    <span id="trigger">
```

Comment:
@semujju we agreed to remove all triggers and use the controls tag based on values just like in ugandaemr_iss

Reply:
done